frank/geometry.py

```diff
@@ -339,6 +344,9 @@ def fit(self, u, v, V, weights):
             Weights on the visibilities
         """
 
+        if self._inc_pa:
```

Comment:
Can we have a check to ensure that we are fitting something? i.e. one of inc_pa and phase_centre is None. Same for FitGeometryFourierBessel

Reply:
Good point, fixed in 1eae50b